components/Banner/BannerSettings.php

```diff
@@ -31,28 +31,97 @@ public function settings()
     public function settingsFields($section)
     {
         add_settings_field(
-            'banner_title',
-            __('Banner title', 'cookie-compliance-for-wordpress'),
-            [$this, 'bannerTitle'],
+            'ga_analytics_id',
+            __('GA ID', 'cookie-compliance-for-wordpress'),
+            [$this, 'setGoogleAnalyticsID'],
+            'cookie-compliance-for-wordpress-settings',
+            $section
+        );
+        add_settings_field(
+            'is_using_youtube',
+            __('I am not using YouTube', 'cookie-compliance-for-wordpress'),
+            [$this, 'isUsingYouTube'],
+            'cookie-compliance-for-wordpress-settings',
+            $section
+        );
+        add_settings_field(
+            'is_using_twitter',
+            __('I am not using Twitter', 'cookie-compliance-for-wordpress'),
+            [$this, 'isUsingTwitter'],
+            'cookie-compliance-for-wordpress-settings',
+            $section
+        );
+        add_settings_field(
+            'is_using_vimeo',
+            __('I am not using Vimeo', 'cookie-compliance-for-wordpress'),
+            [$this, 'isUsingVimeo'],
             'cookie-compliance-for-wordpress-settings',
             $section
         );
     }
 
     /**
-     * Add custom banner title
+     * Function that collects inputed GA tracking ID and running checks on it.
      */
-    public function bannerTitle()
+    public function setGoogleAnalyticsID()
     {
-
         $options = get_option('ccfw_plugin_settings');
-        $bannerTitle = $options['banner_title'] ?? '';
+        $googleAnalyticsID = $options['ga_analytics_id'] ?? '';
 
         ?>
-        <input type='text' name='ccfw_plugin_settings[banner_title]'
-        placeholder="Add a custom cookie banner title" value='<?php echo esc_attr($bannerTitle); ?>'
+        <input type='text' name='ccfw_plugin_settings[ga_analytics_id]'
```

Comment:
Did we want to add a sentence here like before? e.g. I guess we know what its for. I wondered wether we wanted to keep this on the dashboard so the user does. e.g. 'Enter your GA tracking ID and save. This enables the banner to disable/enable your GA tracking for visitors.'

Reply:
Any reason for getting rid of the editable title field? Was it seen as too risky in that people might add in bad titles?

Reply:
I put it in because at one point it got mistaken for the GA field, which lead to the GA code being displayed on a site rather than a banner title. So I thought I'd simplify the banner to the core essentials for now, whilst we get it doing everything we need :)